assets/sass/atoms/_overlays.scss

```diff
@@ -11,4 +11,24 @@
     right: 0px;
     background: rgba(255, 255, 255, 0.8);
   }
+  .fullscreen{
+    z-index: 100000;
+    position: fixed;
+    top: 0px;
+    left: 0px;
+    width: 100%;
+    height: 100%;
+    background: rgba(34, 34, 34, 0.8) none repeat scroll 0% 0%;
+    .btn-close{
+      z-index: 100001;
+      position: absolute;
+      top: 0px;
+      background: rgb(255, 0, 0) none repeat scroll 0% 0%;
+      color: rgb(255, 255, 255);
+      text-decoration: none;
+      font-size: 1.4em;
+      padding: 3px 14px;
+      right: 0px;
```

Comment:
It's nicer to group all similar property. So here, `right` with `top` and `position`